### ISSUE TITLE
RFID Karten beschreiben

### ISSUE THREAD

**Issue by raph-ael** [raph-ael/tonuino-toolbox#2]
Die Funktion die Karten zu beschreiben muss unbedingt rein. Stellt sich die Frage über welchen weg die Smartphone-App der Toolbox kommuniziert. Bluetooth, Netzwerk, USB, Internet... gibt einige Möglichkeiten und wer hat sollte auch ein USB-Gerät nutzen können.

**Comment by xfjx** [comment]
Hi, eventuell könnten man auch eine Funktion in die TonUINO Firmware bauen und dein Tool würde einfach über USB (als serielle Schnittstelle) mit der Box sprechen und das RFID-Modul des TonUINOs zum programmieren der Karten verwendet.

**Comment by raph-ael** [comment]
Ja, klar das wäre deluxe, bei meiner ersten Box ist es zwar etwas unkomfortabel an den USB-Anschluss zu kommen, aber wenn man das beim bau schon einplant ist es natürlich sehr Benutzerfreundlich.

Es gibt auf jedenfall eine gut gepflegte Bibliothek für nodejs (https://serialport.io/) damit müsste es auch für alle Betriebssystem relativ leicht sein den Port anzusprechen, Der User müsste ggf. den Port abändern können, aber wenn die Treiber richtig installiert sind, kann auch automatisch erkannt werden welcher port der arduino ist.

Was denkst Du wie würde am TonUINO das Handling sein. 
* Eine Option im Admin Menü die den Karten-beschreibe-modus aktiviert
* dann Karte auflegen
* mit einem klick sendet das Tool was auf der karte landen soll..
